<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageInfoForm</class>
 <widget class="QWidget" name="ImageInfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <item row="1" column="1">
    <widget class="QLineEdit" name="le_format">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="CheckBoxReadOnly" name="cb_isGrayScale">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="CheckBoxReadOnly" name="cb_isNull">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Is Null</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Has Alpha channel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Is Grayscale</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="CheckBoxReadOnly" name="cb_hasAlpha">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Format</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckBoxReadOnly</class>
   <extends>QCheckBox</extends>
   <header>CustomWidget/CheckBoxReadOnly.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
